cmake/External/ck.cmake

```diff
@@ -0,0 +1,60 @@
+#
+# create INTERFACE target for CK library
+#
+if(NOT __ck_kernels_included)
+  set(__ck_kernels_included TRUE)
+
+  set(ck_kernels_install_dir "${PROJECT_SOURCE_DIR}/torch/lib")
+
+  set(__ck_kernels_version 0.1)
+
+  # create INTERFACE target
+  add_library(__ck_lib INTERFACE)
+
+  if(DEFINED ENV{CK_KERNELS_INSTALLED_PREFIX})
+    # Copy .so from $ENV{CK_KERNELS_INSTALLED_PREFIX} into ${ck_kernels_install_dir}
+    install(DIRECTORY
+            $ENV{CK_KERNELS_INSTALLED_PREFIX}/
+            DESTINATION ${ck_kernels_install_dir}
+            FILES_MATCHING_PATTERN "*.so")
+    message(STATUS "Using Preinstalled CK_kernels from $ENV{CK_KERNELS_INSTALLED_PREFIX}; installed at ${ck_kernels_install_dir}")
+  elseif(DEFINED ENV{CK_KERNELS_PACKAGE_BASE_URL})
```

Comment:
@alugorey @pruthvistony @jeffdaily Does this solution work to keep the internal package url from being exposed? I was able to build PyTorch successfully and run test_transformers tests with CK backend. Caveat is that anyone wanting to build `release/2.6` branch from source will need to set this env var, unless we set `USE_CK_FLASH_ATTENTION` to `OFF` by default: https://github.com/ROCm/pytorch/pull/2031/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR869 ... Maybe we should?